README.md

```diff
@@ -1,27 +1,32 @@
 ![](https://github.com/CESNET/perun-wui/workflows/Maven%20build/badge.svg?branch=master)
 
 <a href="https://perun-aai.org"><img style="float: left; position: relative;" src="https://raw.githubusercontent.com/CESNET/perun/master/perun-web-gui/src/main/webapp/img/logo.png"></a>
-## Perun WUI ##
+## Perun WUI
 
-### Web interface for Perun using GWT and Bootstrap3 ###
+[![conventional commits](https://img.shields.io/badge/semantic--release-conventional-e10079?logo=semantic-release)](https://www.conventionalcommits.org/)
+[![commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](https://commitizen.github.io/cz-cli/)
+[![CI status](https://github.com/cesnet/perun-wui/actions/workflows/semantic-release.yml/badge.svg)](https://github.com/cesnet/perun-services/actions/workflows/semantic-release.yml)
```

Comment:
```suggestion
[![CI status](https://github.com/CESNET/perun-wui/workflows/Maven%20build/badge.svg?branch=master)](https://github.com/cesnet/perun-services/actions/workflows/maven.yml)
```

Reply:
```suggestion
[![CI status](https://github.com/CESNET/perun-wui/workflows/test%20build/badge.svg?branch=master)](https://github.com/CESNET/perun-wui/actions/workflows/maven.yml)
```

The `test build` should be used, because the Maven (as a name) workflow does not exist.

Reply:
This is the correct image source: `https://github.com/CESNET/perun-wui/workflows/test%20build/badge.svg?branch=master`

which will render as:

![test](https://github.com/CESNET/perun-wui/workflows/test%20build/badge.svg?branch=master)

And this is the correct link: [https://github.com/CESNET/perun-wui/actions/workflows/maven.yml](https://github.com/CESNET/perun-wui/actions/workflows/maven.yml) (click it)

So the correct markdown is:

```md
[![CI status](https://github.com/CESNET/perun-wui/workflows/test%20build/badge.svg?branch=master)](https://github.com/CESNET/perun-wui/actions/workflows/maven.yml)
```

which will render as:

[![CI status](https://github.com/CESNET/perun-wui/workflows/test%20build/badge.svg?branch=master)](https://github.com/CESNET/perun-wui/actions/workflows/maven.yml)

Reply:
Yes, it is included in the PR.

Reply:
Sorry, I was looking at a wrong diff, yours is correct